packages/web-components/src/components/va-notification/va-notification.tsx

```diff
@@ -0,0 +1,144 @@
+import { 
+  Component, 
+  Element,
+  Event,
+  EventEmitter,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Notification
+ * @maturityCategory caution
+ * @maturityLevel proposed
+ */
+
+@Component({
+  tag: 'va-notification',
+  styleUrl: 'va-notification.scss',
+  shadow: true,
+})
+export class VaNotification {
+  @Element() el!: any;
+
+  /**
+   * If `true`, the card will be visible.
+   */
+  @Prop() visible?: boolean = true;
+
+  /**
+   * Symbol indicates type of notification 
+   * Current options are: action-required, update
+   */
+  @Prop() symbol?: string = 'none';
+
+  /**
+   * Aria-label text for the close button.
+   */
+  @Prop() closeBtnAriaLabel?: string = 'Close notification';
+
+  /**
+   * If `true`, a close button will be displayed.
+   */
+  @Prop({ reflect: true }) closeable?: boolean = false;
+
+  /**
+   * If `false`, card will not have border
+   */
+  @Prop({ reflect: true }) hasBorder?: boolean = true;
+
+  /**
+   * If `false`, card will not have the word "CLOSE" next to close icon
+   */
+  @Prop() hasCloseText?: boolean = false;
+
+  /**
+   * Headline for notification
+   */
+  @Prop() headline?: string;
+
+  /**
+   * Define level for headline. Font size will remain the same regardless of header level.
+   */
+  @Prop() headlineLevel?: string = '3';
+
+  /**
+   * Destination URL for link (optional)
+   */
+  @Prop() href?: string;
+
+  /**
+   * Text for destination link. Set to empty string if you don't want a link.
+   */
+  @Prop() text?: string;
+  
+  /**
+   * Fires when the component is closed by clicking on the close icon. This fires only
+   * when closeable is true.
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  closeEvent: EventEmitter;
+
+  private closeHandler(e: MouseEvent): void {
+    this.closeEvent.emit(e);
+  }
+
+  private getHeadlineLevel() {
+    const number = parseInt(this.headlineLevel, 10);
+    return number >= 1 && number <= 6 ? `H${number}` : null;
```

Comment:
If `headlineLevel` is a non-numeric string, this would result in `null` being returned, which would prevent a headline from displaying. Is this the desired effect? Would it be better to return a `h3` by default and have L#119 below show or not-show based on the value of the `headline` prop?